### ISSUE TITLE
Accessibility: Allow image descriptions to be added/edited after posting

### ISSUE THREAD

**Issue by CorgiDev** [mastodon/mastodon#22423]
### Pitch

Currently, images get a border and notification placed on them when they do not have descriptive text, but there is no way to edit the description or add one to the images once the post has been submitted. 

You have to delete the image and reattach it to the post to get this option. This could be very difficult if you happen to be on a different device from the one where you initially made the post (one that does not have the image on it), or if you are on mobile since then you cannot as easily download the image from the post and reattach it that way. This could deter people from going back and fixing the lack of descriptive text when we should be trying to encourage such accessibility.

### Motivation

Primarily users who use screen readers, but in the end accessibility improvements benefit everyone.

**Comment by ineffyble** [comment]
Duplicate of #20887

**Comment by CorgiDev** [comment]
Thanks for linking to the other bug. 

Sorry for making a duplicate issue. I tried to search first, but it didn't come up at first. 

Glad to see it's already being looked at. Also, I love the current way Mastodon notifies you that you didn't add alt text. Absolutely, made my day to see that.
